viite-UI/src/view/RoadAddressChangesBrowserWindow.js

```diff
@@ -0,0 +1,331 @@
+(function (root) {
+    root.RoadAddressChangesBrowserWindow = function (backend) {
+        const MAX_ROWS_TO_DISPLAY = 100;
+        const MAX_YEAR_PARAM = 2050;
+        const MIN_YEAR_PARAM = 1900;
+        const MAX_LENGTH_FOR_ROAD_NUMBER = 5;
+        const MAX_LENGTH_FOR_ROAD_PART_NUMBER = 3;
+        let searchParams = {};
+        let datePickerStartDate = '';
+        let datePickerEndDate = '';
+        const me = this;
+
+        const roadAddressChangesBrowserWindow = $('<div class="form-horizontal road-address-changes-browser-window"></div>').hide();
+        roadAddressChangesBrowserWindow.append('<a href="" target="_blank">' + // TODO add link to manual, once manual is done
+            '<button class="btn-manual" title="Avaa käyttöohje">' +
+            '<i class="fas fa-question"></i>' +
+            '</button>' +
+            '</a>');
+        roadAddressChangesBrowserWindow.append('<button class="close btn-close">x</button>');
+        roadAddressChangesBrowserWindow.append('<div class="road-address-browser-modal-header">Tieosoitemuutosten katselu</div>');
+        roadAddressChangesBrowserWindow.append('' +
+            '<form id="roadAddressChangesBrowser" class="road-address-browser-form">' +
+            '<div class="input-container"><label class="control-label-small">Alkupvm</label> <input type="text" class="road-address-browser-date-input" id="roadAddrChangesStartDate" value="' + getCurrentDate() + '" required/></div>' +
+            '<div class="input-container"><label class="control-label-small">Loppupvm</label> <input type="text" class="road-address-browser-date-input" id="roadAddrChangesEndDate" value="" /></div>' +
+            '<div class="input-container">' +
+                '<label class="control-label-small">Ely</label>' +
+                '<select name id="roadAddrChangesInputEly" /> ' +
```

Comment:
...ja äänestäisin tän elydropdownin säätämistä johonkin "komponenttikirjstoon" kans

Reply:
Hyym ihan devil advocate, mitä lisää se tuo. onko tulossa näitä samanlaisa enemmänkin? 
Sellainen geneerinen on hieno mutta pitäisi olla varma että on enemmä käyttötapauksia kuin yksi.
Ehkä kolmannessa samanlaisen vaatimuksen kohdalla vois miettiä sitä geneeristä mutta sitten on se refactorointi homma.